types/whatwg-url/lib/URLSearchParams.d.ts

```diff
@@ -0,0 +1,66 @@
+import { Options as WebIDLConversionOptions } from 'webidl-conversions';
+import { implementation as URLSearchParamsImpl } from './URLSearchParams-impl';
+
+/**
+ * Checks whether `obj` is a `URLSearchParams` object with an implementation
+ * provided by this package.
+ */
+export function is(obj: unknown): obj is URLSearchParams;
+
+/**
+ * Checks whether `obj` is a `URLSearchParamsImpl` WebIDL2JS implementation object
+ * provided by this package.
+ */
+export function isImpl(obj: unknown): obj is URLSearchParamsImpl;
+
+/**
+ * Converts the `URLSearchParams` wrapper into a `URLSearchParamsImpl` object.
+ *
+ * @throws {TypeError} If `obj` is not a `URLSearchParams` wrapper instance provided by this package.
+ */
+export function convert(obj: unknown, options?: WebIDLConversionOptions): URLSearchParamsImpl;
+
+/**
+ * Creates a new `URLSearchParams` instance.
+ *
+ * @throws {Error} If the `globalObject` doesn't have a WebIDL2JS constructor
+ *         registry or a `URLSearchParams` constructor provided by this package
+ *         in the WebIDL2JS constructor registry.
+ */
+export function create(
+	globalObject: object,
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/whatwg-url/lib/URLSearchParams.d.ts;
  Line: 31;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.